getting-started/pom.xml

```diff
@@ -46,6 +46,22 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+
```

Comment:
Can it be removed?
If not, can a comment be added to explain why this dependency is there but commented.